source/mrmeshpy/MRPythonBaseExposing.cpp

```diff
@@ -149,6 +148,15 @@ MR_ADD_PYTHON_CUSTOM_DEF( mrmeshpy, name, [] ( pybind11::module_& m ) \
         }, pybind11::keep_alive<0, 1>() );\
     m.def( "dot", ( type( * )( const VectorType&, const VectorType& ) )& MR::dot<type>, pybind11::arg( "a" ), pybind11::arg( "b" ), "dot product" );\
     m.def( "cross", ( type( * )( const VectorType&, const VectorType& ) )& MR::cross<type>, pybind11::arg( "a" ), pybind11::arg( "b" ), "cross product" );\
+    \
+    []<typename T = type, typename V = VectorType>{ \
+        if constexpr (std::is_floating_point_v<T>) \
+        { \
+            MR_PYTHON_CUSTOM_CLASS( name ) \
+                .def( "normalized", &V::normalized ) \
+            ; \
+        } \
+    }(); \
```

Comment:
why wrap in lambda?

Reply:
Because `if constexpr` doesn't do its magic when the condition doesn't depend on a template parameter. Have to make a template lambda. Will add a comment.